pages/agent/clusters.md

```diff
@@ -90,6 +90,23 @@ To add a maintainer to a cluster:
 1. Select a user or team.
 1. Click _Add Maintainer_.
 
+### Pause a queue
+
+You can pause a queue to prevent jobs from being dispatched to agents associated with that queue.
+To pause a queue:
+
+1. Navigate to your cluster’s _Queues_.
+1. Select on the queue you wish to pause.
+1. Select _Edit_.
+1. Under _Queue Management_, select _Pause Queue_.
+1. Enter an optional note in the dialog if needed, and confirm that you wish to pause the queue. The note will be displayed on the queue page, and any affected builds.
+
+Jobs which have _already_ been dispatched to agents in the queue prior to pausing will continue to run. New jobs which target the paused queue will 'wait' until the queue is resumed.
+
+Trigger steps do not rely on agents, so they will run unless they have a dependency that has been halted by the paused queue. The behaviour of the jobs they trigger depends on their configuration. If a triggered job targets the paused queue, it will 'wait' until the queue is resumed. If a triggered job does not target the paused queue, it will run as usual.
```

Comment:
Also a good clarifying note!

Just took the quotes off "wait" here too:

```suggestion
Trigger steps do not rely on agents, so they will run unless they have a dependency that has been halted by the paused queue. The behaviour of the jobs they trigger depends on their configuration. If a triggered job targets the paused queue, it will wait until the queue is resumed. If a triggered job does not target the paused queue, it will run as usual.
```

My only other thought here is whether "halted" is the right term. We're not using it anywhere else in this section. I wonder if we could rephrase that first sentence to avoid it...

> Trigger steps do not rely on agents, so they will run unless they have dependencies that are also waiting with the paused queue.

Maybe?

Reply:
+1 to avoiding adding a new term here if we can avoid it. Another option for tightening:

> Trigger steps do not rely on agents, so they will run unless they have dependencies waiting on the paused queue.